MAUI/MauiAppWithBroker/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,121 @@
+{
+  "Sample": {
+    "Title": "A .NET MAUI MAUI using MSAL.NET and a broker to acquire an access token and call Microsoft Graph API",
+    "Level": 200,
+    "Client": "MAUI (iOS, Android, UWP)",
+    "RepositoryUrl": "active-directory-xamarin-native-v2",
+    "Endpoint": "AAD v2.0",
+    "Description": "A .NET MAUI app using MSAL.NET to sign-in users using a broker and acquiring a token to call Microsoft Graph Api",
+    "Service": "Microsoft Graph",
+    "Languages": [
+      "csharp"
+    ],
+    "Products": [
+      "maui",
+      "azure-active-directory"
+    ],
+    "Library": [
+      "Microsoft Graph"
+    ],
+    "Platform": "MAUI"
+  },
+  "ReadmeAboutTheCode": {
+    "IncludeFilePath": "../ReadmeFiles/AboutTheCode.md"
+  },
+  "ReadmeTroubleshooting": {
+    "IncludeFilePath": "../ReadmeFiles/TroubleShooting.md"
+  },
+  "ReadmeNextSteps": {
+    "IncludeFilePath": "../ReadmeFiles/NextSteps.md"
+  },
+  /*
+    This section describes the Azure AD Applications to configure, and their dependencies
+  */
+  "AADApps": [
+    {
+      "Sample": {
+        "SampleSubPath": "MAUI/MauiAppWithBroker"
+      },
+      "Id": "client",
+      "Name": "active-directory-maui-with-broker-v2",
```

Comment:
>active-directory-maui-with-broker-v2 [](http://example.com/codeflow?start=15&length=36)

consider adding a character or two so that the app I created earlier in the basic sample does not get overwritten